visualizer/Core/RecordingAnalyticsDrawer/Views/RecordingAnalyticsDrawerView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  RecordingAnalyticsDrawerView.swift
+//  visualizer
+//
+//  Created by Yeung on 13/7/2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct RecordingAnalyticsDrawerView: View {
+    @EnvironmentObject var recordingAnalyticsDrawerVM: RecordingAnalyticsDrawerViewModel
+    @EnvironmentObject var audioVM: AudioViewModel
+    @Binding var isShowing: Bool
+    @Binding var isShowingModal: Bool
+    
+    let onClose: () -> Void
+    
+    var body: some View {
+        DrawerView(isShowing: $isShowing, isShowingModal: $isShowingModal) {
+            VStack(alignment: .leading) {
+                ZStack(){
+                    HStack(){
+                        Spacer()
+                        Text("Analytics")
+                            .font(.system(size: 22))
+                            .bold()
+                        Spacer()
+                    }
+                    HStack(){
+                        Spacer()
+                        CloseButton(action: onClose)
+                            .padding([.top, .bottom, .trailing])
+                    }
+                }
+                HStack(){
+                    Spacer()
+                    Button("Notes", action: {
+                        recordingAnalyticsDrawerVM.recordingAnalyticsDrawer.selected = RecordingAnalyticsDrawer.AnalyticsTypes.notes
+                    })
+                    Button("Melody", action: {
+                        recordingAnalyticsDrawerVM.recordingAnalyticsDrawer.selected = RecordingAnalyticsDrawer.AnalyticsTypes.melody
+                    })
+                    Spacer()
+                }
+                
+                Amplitudes()
+                if (recordingAnalyticsDrawerVM.isSelected(RecordingAnalyticsDrawer.AnalyticsTypes.notes)){
```

Comment:
Hmm, this should be the iOS style segmented control. Which can be implemented easily with some native components.

You can take a look at line 86 on `TimbreView.swift`
```swift
                    Picker("Display Mode", selection: $displayMode) {
                        Text(DisplayMode.yours.title)
                            .tag(DisplayMode.yours)
                        Text(DisplayMode.suggested.title)
                            .tag(DisplayMode.suggested)
                        Text(DisplayMode.mixed.title)
                            .tag(DisplayMode.mixed)
                    }
                    .pickerStyle(.segmented)
```